src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -181,29 +185,29 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
   }
 
   const consentTypeProperty = (consented: boolean, item: any): JSX.Element => {
```

Comment:
We could probably be clearer by saying what the function does. Does it get/create the consent type property?

Reply:
It should be getConsentTypeProperty

Reply:
I've also renamed the other column item headers with proper action verbs